api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/reshaping/RepeatVector.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.core.layer.reshaping
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.layer.Layer
+import org.jetbrains.kotlinx.dl.api.core.shape.TensorShape
+import org.tensorflow.Operand
+import org.tensorflow.Shape
+import org.tensorflow.op.Ops
+import org.tensorflow.op.core.Tile
+
+/**
+ * Layer that repeats the input [n] times.
+ *
+ * Input shape: `2D tensor of shape (num_samples, features)`.
+ *
+ * Output shape: `3D tensor of shape (num_samples, n, features)`.
+ *
+ * @property n Repetition factor.
+ * @property [name] Custom layer name.
+ * @constructor Creates [RepeatVector] object.
+ *
+ * @author Stan van der Bend
+ * @since 0.2
+ */
+public class RepeatVector(
+    public val n: Int,
+    name: String = ""
+) : Layer(name) {
+
+    override fun build(tf: Ops, kGraph: KGraph, inputShape: Shape) {
+        //left empty
+    }
+
+    override fun computeOutputShape(inputShape: Shape): Shape {
+        require(inputShape.numDimensions() == 2) { "input tensor must have 2 dimensions" }
+        val tensorShape = TensorShape(inputShape)
+        // TODO: maybe make `n` of type Long?
```

Comment:
imo you should as for now most of the ocnfiguration properties seems to have Long type.
btw why not just 
```kotlin
return Shape.make(inputShape[0], n, inputShape[1])
```
the `tensorShape` is unnecessary.

Reply:
`Shape` doesn't have operator functions for get, I could do:
```Kotlin
        val input = inputShape.toLongArray()
        return Shape.make(input[0], n.toLong(), input[1])
```

Reply:
@dosier You can simply use `size`:
```kotlin
return Shape.make(inputShape.size(0), n.toLong(), inputShape.size(1))
```

Reply:
@mkaze ah of course, thanks